### ISSUE TITLE
export-installation logs every second, causing CI build logs to be very large

### ISSUE THREAD

**Issue by camelpunch** [pivotal-cf/om#104]
We left a backup of an OpsMan to run over the weekend, and were disappointed to find that our Concourse logs were unviewable because they filled with lines like:

```
1s elapsed, waiting for response from Ops Manager...
2s elapsed, waiting for response from Ops Manager...
3s elapsed, waiting for response from Ops Manager...
4s elapsed, waiting for response from Ops Manager...
5s elapsed, waiting for response from Ops Manager...
6s elapsed, waiting for response from Ops Manager...
7s elapsed, waiting for response from Ops Manager...
8s elapsed, waiting for response from Ops Manager...
9s elapsed, waiting for response from Ops Manager...
10s elapsed, waiting for response from Ops Manager...
11s elapsed, waiting for response from Ops Manager...
12s elapsed, waiting for response from Ops Manager...
13s elapsed, waiting for response from Ops Manager...
14s elapsed, waiting for response from Ops Manager...
15s elapsed, waiting for response from Ops Manager...
16s elapsed, waiting for response from Ops Manager...
17s elapsed, waiting for response from Ops Manager...
18s elapsed, waiting for response from Ops Manager...
```

ad nauseum.

Can we get a `--quiet` option, or preferably something that tells us about e.g. network issues?

**Comment by acrmp** [comment]
Hi @camelpunch,

We recently pushed some unreleased changes to `master` that provide a `--polling-interval` flag on `export-installation` that might help you here. Recent concourse versions also seem to collapse the waiting for response messages in the build output too.

Was the excessive output because the export actually took a long time, or was there a networking issue that om swallowed?

Thanks,

Andrew.

**Comment by camelpunch** [comment]
In this case it was a network issue that was swallowed. I won't be using `om` anymore, but @simonijones might be interested in this development. Thanks!

**Comment by kkallday** [comment]
Hi @camelpunch and @simonjjones,

In our pipelines we've noticed that tasks sometimes stay open of an extended amount of time. Our solution is to put a `timeout` on the build step (http://concourse.ci/single-page.html#timeout-step).

Ops manager runs a `thin` server that has a timeout of 1800s (https://github.com/pivotal-cf/installation/blob/3d7873a42a3c1edc60be53ce4232c3f3531a483a/web/config/thin.production.yml#L5) so I'd think the connection would be closed on their end. `om` also has a default timeout of 1800s in the httpclient. We believe this might be a concourse issue and we've raised the issue with them.

For the meantime, setting a `timeout` on all potentially long running `om` tasks like deploy, export, import would prevent the issue of build logs filling up your CI.

If this happens again, it would be nice to look at the tcp connection state of the `om` request on the client side and the ops manager side (running `netstat -an` would give you this info). It would also be nice to know how traffic reaches your opsman (for instance is there a load balancer in the path, haproxy, etc). This would help us further debug the issue.

cc @apshirley

- Kevin and @utako
